unicorn_mode/samples/compcov_x64/compcov_test_harness.py

```diff
@@ -19,7 +19,10 @@
 import signal
 
 from unicornafl import *
-from unicornafl.x86_const import *
+
+monkeypatch()
```

Comment:
What is this? :)

Reply:
https://github.com/AFLplusplus/unicornafl#migration To make it work both on ucafl1 and ucafl2

Reply:
We don't need it anymore I think. If you have this file, you'll have unicornafl2?

Reply:
Okay fine, let's remove it.